### ISSUE TITLE
 LHN - Chat disappear from LHN when user haven't sent any message but create Private Note

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#28481]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Navigate to staging.new.expensify.com
2. Click on FAB > Send Message
3. Select User B you never have chat before
4. Don't send any message
5. Create Private Note with this User
6. Navigate to other chat

## Expected Result:
Since User A create Private Note with other User B the conversation should remain in LHN

## Actual Result:
Chat disappear from LHN when user haven't sent any message but create Private Note

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.75-0

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/c8450d78-c8ec-4ad6-9ffd-f49c89233477

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by isabelastisser** [comment]
@burczu can you please review the proposal above? Thanks!

**Comment by burczu** [comment]
@allroundexperts Thanks for detailed [proposal](https://github.com/Expensify/App/issues/28481#issuecomment-1741548583). I've just reviewed and tested it and I think it does identify the issue correctly. Also the provided solution seem to be accurate, so I think we can proceed with it.

🎀 👀 🎀 C+ reviewed

**Comment by robertjchen** [comment]
Great proposal, let's move forward with that fix!

**Comment by burczu** [comment]
@robertjchen @isabelastisser Plase take a look at the comment @allroundexperts posted in his PR: https://github.com/Expensify/App/pull/29096#issuecomment-1753273444

It looks like we need to put this PR on hold, and switch to `Internal` for now to first update the backend. What do you think?

**Comment by isabelastisser** [comment]
Bump @robertjchen can you see the comment above? Do we ned to put this on hold? thanks!

**Comment by robertjchen** [comment]
Yes, I would agree- let's put it on Hold for now while we look into the `Internal` aspect.

**Comment by robertjchen** [comment]
Still on hold for now

**Comment by burczu** [comment]
Hey @isabelastisser! I'm leaving C+ role today and joining Waves. Please assign another C+ instead of me. Thank you!
